web/adelacecilia/html/div tag/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <title>The "div" tag</title>
+<head>
+<style>
+.myDiv {
+  border: 7px outset #20d120;
+  background-color: black;    
+  text-align: center;
+  color: #20d120;
+}
+</style>
+</head>
+<body>
+
+<h1>The "div" element</h1>
```

Comment:
Please dont leave empty lines in your HTML :)

Reply:
please delete blank line 15